src/modules/powerrename/ui/PowerRenameUI.h

```diff
@@ -139,6 +157,8 @@ class CPowerRenameUI :
     DWORD m_currentRegExId = 0;
     UINT m_selectedCount = 0;
     UINT m_renamingCount = 0;
+    UINT m_initialDPI = 0;
+    DialogItemsPositioning m_itemsPositioning;
```

Comment:
let's zero-initialize the diffs.
```suggestion
    DialogItemsPositioning m_itemsPositioning {};
```